eclipse/che
# WIP - Adding npm package versioning

### What does this PR do?
Add npm package versioning.


### What issues does this PR fix or reference?
Publish che-e2e npm package into public registry https://github.com/eclipse/che/issues/18772
Have proper versioning of e2e npm package https://github.com/eclipse/che/issues/18771


### How to test this PR?
To verify that dependencies are not broken, run any test.
To verify package version setup, first set env var `export NPM_AUTH_TOKEN=aaa` and  run `mvn install`.


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
